src/main/java/frc/robot/Robot.java

```diff
@@ -28,6 +45,31 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+
+    camera1 = CameraServer.startAutomaticCapture(0);
+    camera2 = CameraServer.startAutomaticCapture(1);
+    // Tell both cameras to stream to the dashboard
+    camera1.setConnectionStrategy(ConnectionStrategy.kKeepOpen);
+    camera2.setConnectionStrategy(ConnectionStrategy.kKeepOpen);
+
+    cameraSelection = NetworkTableInstance.getDefault().getTable("").getEntry("CameraSelection");
+
+    // this is to put git info in the dashboard & Logs
+    String deployDir = Filesystem.getDeployDirectory().getPath();
+    String branchName = "unknown";
+    String commitHash = "unknown";
+    try {
+      branchName = Files.readString(Path.of(deployDir, "branch.txt"));
+      commitHash = Files.readString(Path.of(deployDir, "commit.txt"));
+
+    } catch (IOException e) {
+      e.printStackTrace();
+      System.out.println("Parsing Git metadata Files Failed");
+    }
+    System.out.println("Branch: " + branchName);
+    System.out.println("Commit: " + commitHash);
+    SmartDashboard.putString("Branch", branchName);
+    SmartDashboard.putString("Commit", commitHash);
```

Comment:
Nit, lets put it in a separate function with a proper docstring to keep things clean